TraitsExpanded/TraitSystemBehavior.cs

```diff
@@ -1,16 +1,20 @@
-﻿using System.Collections.Generic;
+﻿using System;
+using System.Collections.Generic;
 using System.Linq;
 using TaleWorlds.CampaignSystem;
 using TaleWorlds.CampaignSystem.SandBox.CampaignBehaviors;
 using TaleWorlds.Core;
 using TaleWorlds.Engine;
+using TraitsExpanded.TraitSets;
 
 namespace TraitsExpanded
 {
     public class TraitSystemBehavior : CampaignBehaviorBase
     {
-        public static readonly List<ITraitSet> AllTraitSets = new List<ITraitSet>();
-        public static Dictionary<CharacterObject, List<ITraitSet>> CharacterTraitInfo = new Dictionary<CharacterObject, List<ITraitSet>>();
+        public static readonly List<TraitSet> AllTraitSets = new List<TraitSet>();
```

Comment:
we shouldn't expose these members publicly, I don't think

Reply:
Yup, will redo the access modifiers once I get the pipeline down